src/main/java/io/github/mzmine/modules/visualization/chromatogramandspectra/ChromatogramAndSpectraVisualizer.java

```diff
@@ -671,15 +663,14 @@ public boolean equals(Object o) {
       return false;
     }
     ChromatogramAndSpectraVisualizer that = (ChromatogramAndSpectraVisualizer) o;
-    return showSpectraOfEveryRawFile == that.showSpectraOfEveryRawFile &&
-        chromPlot.equals(that.chromPlot) &&
-        spectrumPlot.equals(that.spectrumPlot) &&
-        Objects.equals(scanSelection.get(), that.scanSelection.get()) &&
-        Objects.equals(mzRange.get(), that.mzRange.get()) &&
-        Objects.equals(chromPosition.get(), that.chromPosition.get()) &&
-        Objects.equals(rtMarker, that.rtMarker) &&
-        chromMzTolerance.get().equals(that.chromMzTolerance.get()) &&
-        Objects.equals(filesAndDataSets, that.filesAndDataSets);
+    return showSpectraOfEveryRawFile == that.showSpectraOfEveryRawFile
```

Comment:
Sometimes i wonder why the style is different although we use the same formatter

Reply:
no clue, weird....